hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -355,28 +357,80 @@ export class MissingContainers extends React.Component<Record<string, object>, I
           <Tabs defaultActiveKey='1'>
             <TabPane key='1' tab="Missing">
               <Table
-                expandRowByClick dataSource={missingDataSource} columns={CONTAINER_TAB_COLUMNS}
+                expandRowByClick dataSource={missingDataSource} 
+                columns={CONTAINER_TAB_COLUMNS.reduce<any[]>((filtered, column) => {
+                  if (column.isSearchable) {
+                    const newColumn = {
+                      ...column,
+                      ...new ColumnSearch(column).getColumnSearchProps(column.dataIndex)
+                    };
+                    filtered.push(newColumn);
+                  } else {
+                    filtered.push(column);
+                  }
+
+                  return filtered;
+                }, [])}
                 loading={loading}
                 pagination={paginationConfig} rowKey='containerID'
                 expandedRowRender={this.expandedRowRender} onExpand={this.onRowExpandClick}/>
             </TabPane>
             <TabPane key='2' tab='Under-Replicated'>
               <Table
-                expandRowByClick dataSource={underReplicatedDataSource} columns={CONTAINER_TAB_COLUMNS}
+                expandRowByClick dataSource={underReplicatedDataSource}
+                columns={CONTAINER_TAB_COLUMNS.reduce<any[]>((filtered, column) => {
+                  if (column.isSearchable) {
+                    const newColumn = {
+                      ...column,
+                      ...new ColumnSearch(column).getColumnSearchProps(column.dataIndex)
+                    };
+                    filtered.push(newColumn);
+                  } else {
+                    filtered.push(column);
+                  }
+
+                  return filtered;
+                }, [])}
                 loading={loading}
                 pagination={paginationConfig} rowKey='containerID'
                 expandedRowRender={this.expandedRowRender} onExpand={this.onRowExpandClick}/>
             </TabPane>
             <TabPane key='3' tab='Over-Replicated'>
               <Table
-                expandRowByClick dataSource={overReplicatedDataSource} columns={CONTAINER_TAB_COLUMNS}
+                expandRowByClick dataSource={overReplicatedDataSource}
+                columns={CONTAINER_TAB_COLUMNS.reduce<any[]>((filtered, column) => {
+                  if (column.isSearchable) {
+                    const newColumn = {
+                      ...column,
+                      ...new ColumnSearch(column).getColumnSearchProps(column.dataIndex)
```

Comment:
just curious~ if in the future want to enable other columns to be searchable,
do we have to _new ColumnSearch_ object for each column here as well?

Reply:
Just  need to use this isSearchable: true line in columns.

Reply:
Got it! Thanks! And it'd be nice (maybe in another jira) if we could pull out the common parts between Missing, Under-Replicated, Over-Replicated, and Mis-Replicated and reuse!

Reply:
I agree with @DaveTeng0, as I checked the only difference between the `Table`s inside the `TabPane`s are the `dataSource`. Maybe we could create a method with one parameter (`dataSource`) inside this file which creates this `Table` with the given `dataSource`.